backend/coreapp/tests/test_preset.py

```diff
@@ -53,6 +72,54 @@ def test_create_preset_not_authenticated(self) -> None:
         except AssertionError:
             pass
 
+    def test_user_create_preset(self) -> None:
```

Comment:
these tests are great! could you add one to make sure anons can't make presets?

Reply:
There is already one which was already defined, I kept it.

https://github.com/decompme/decomp.me/blob/eca8cd2229974f7482ae77c0edff4a4786890d03/backend/coreapp/tests/test_preset.py#L66-L73